demos/supabase-todolist/lib/attachments/queue.dart

```diff
@@ -54,10 +56,11 @@ class PhotoAttachmentQueue extends AbstractAttachmentQueue {
   }
 
   @override
-  Future<Attachment> deletePhoto(String photoId) async {
```

Comment:
Do we need the breaking change of renaming `deletePhoto` etc? Could we not preserve this method and call the generic `xxxFile` method under the hood?

Reply:
Since this is still in Beta I would rather remove it to avoid confusion for new devs coming in. I think we have at most  a handful of users using this (based on support queries). But I don't feel strongly about this.

Reply:
I'm hesitant to include breaking changes in minor package version bumps. This could cause headaches for existing users.

Reply:
Isn't that the point of version 0 though so people know if there's a minor upgrade it could have breaking changes so should rather use ~ instead of ^?

Reply:
Yeah, you're right. I forgot about the magic of version `0`.